packages/components/aside/src/aside.css

```diff
@@ -0,0 +1,144 @@
+/* General........................................................ */
+
+.aside-container {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	right: 0;
+	left: auto;
+	display: flex;
+	flex-direction: column;
+	min-height: 100vh;
+	width: var(--ts-aside);
+	background-color: var(--ts-color-white);
+	box-shadow: 0 0 3px var(--ts-shadow-color);
+	z-index: var(--ts-zindex-aside);
+}
+
+.aside-container > * {
+	padding: var(--ts-unit);
+}
+
+/* Header......................................................... */
+
+header {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Check this potential box model size issue. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Abox-model)